core/Cargo.toml

```diff
@@ -6,102 +6,102 @@ edition = "2021"
+frame-support = { workspace = true, default-features = true }
+pallet-babe = { workspace = true, default-features = true }
+frame-benchmarking = { workspace = true, default-features = true }
 
 # Polkadot dependencies
-polkadot-core-primitives = { workspace = true }
-polkadot-node-primitives = { workspace = true }
-polkadot-primitives = { workspace = true }
-polkadot-parachain-primitives = { workspace = true }
-polkadot-runtime-parachains = { workspace = true }
-polkadot-service = { workspace = true }
-polkadot-overseer = { workspace = true }
+polkadot-core-primitives = { workspace = true, default-features = true }
+polkadot-node-primitives = { workspace = true, default-features = true }
+polkadot-primitives = { workspace = true, default-features = true }
+polkadot-parachain-primitives = { workspace = true, default-features = true }
+polkadot-runtime-parachains = { workspace = true, default-features = true }
+polkadot-service = { workspace = true, default-features = true }
+polkadot-overseer = { workspace = true, default-features = true }
 ## Currently only needed to make cumulus-relay-chain-inprocess-interface compile, not sure why cumulus works though
-polkadot-cli = { workspace = true }
+polkadot-cli = { workspace = true, default-features = true }
 
 # Cumulus dependencies
-cumulus-primitives-parachain-inherent = { workspace = true }
-cumulus-relay-chain-interface = { workspace = true }
-cumulus-primitives-core = { workspace = true }
-cumulus-client-parachain-inherent = { workspace = true }
+cumulus-primitives-parachain-inherent = { workspace = true, default-features = true }
+cumulus-relay-chain-interface = { workspace = true, default-features = true }
+cumulus-primitives-core = { workspace = true, default-features = true }
+cumulus-client-parachain-inherent = { workspace = true, default-features = true }
 
 [dev-dependencies]
-polkadot-test-runtime = { workspace = true }
+polkadot-test-runtime = { workspace = true, features = ["std"] }
 
-staging-parachain-info = { workspace = true }
+staging-parachain-info = { workspace = true, features = ["std"] }
 
-sp-consensus-beefy = { workspace = true }
-frame-benchmarking = { workspace = true }
-pallet-timestamp = { workspace = true }
-pallet-balances = { workspace = true }
+sp-consensus-beefy = { workspace = true, features = ["std"] }
+frame-benchmarking = { workspace = true, features = ["std"] }
+pallet-timestamp = { workspace = true, features = ["std"] }
+pallet-balances = { workspace = true, features = ["std"] }
 sc-keystore = { workspace = true }
-pallet-transaction-payment = { workspace = true }
+pallet-transaction-payment = { workspace = true, features = ["std"] }
 tracing-subscriber = { workspace = true }
-pallet-aura = { workspace = true }
-fudge-test-runtime = { workspace = true }
-pallet-session = { workspace = true }
-pallet-grandpa = { workspace = true }
-pallet-im-online = { workspace = true }
+pallet-aura = { workspace = true, features = ["std"] }
```

Comment:
If you can to unify everything, you can use here also `default-features = true` instead of `std`